integration

```diff
@@ -903,6 +903,7 @@
   "viragelabs/virage_dashboard",
   "vlumikero/home-assistant-securitas",
   "vmakeev/huawei_mesh_router",
+  "Vova-SH/termux-api",
```

Comment:
PR is not editable.
You need to adjust your branch settings.

Reply:
I created new PR with other source branch: https://github.com/hacs/default/pull/2280
It's ok?

Reply:
It is not, it still does not have maintainer permissions.

Reply:
Oh, sorry. I checked "Allow edits by maintainers" now.